### PR TITLE
Correct docs for R6 classes

### PR DESCRIPTION
**Pull request by olivroy**
See https://rstudio.github.io/chromote/reference/Chromote.html
There is a bug on the top of the page.

This correction is done to circumvent r-lib/roxygen2#1504

Edit: changes to the R files could be reverted once the issue is fixed.

### CODE REVIEW

**Comment by gadenbuie** [comment]
Thanks for pointing this out to us @olivroy. I understand this doesn't look as good as it possibly could, but it sounds like a fix will be applied upstream. As long as our documentation is still readable and functional, I'd prefer to just wait for the upstream fix at which point this will resolve itself.

**Comment by hadley** [comment]
@gadenbuie it's probably going to be a while until a fix gets into roxygen2, so I think you should merge this PR.

File: R/chromote.R [hunk 1]
```diff
@@ -1,7 +1,9 @@
 #' Chromote class
 #'
+#' @description
 #' This class represents the browser as a whole.
 #'
+#' @details
```
Comment (hadley) [inline R/chromote.R:6]: IMO you could remove line 3 and make lines 7-10 the description. (Or even the whole text block could arguably be the description)

**Review by gadenbuie — APPROVED** [review]
Thank you @olivroy! 

@wch or @hadley, I'll let you handle merging since you're actively working on chromote.

File: R/chromote.R (hunk 1 above)
Comment (olivroy) [inline R/chromote.R:6]:
```suggestion
#' @description
```

The first sentence is not very useful indeed.

**Comment by olivroy** [comment]
Only need to redocument and this would be ok to go

**Pull request merged**